src/main/java/apiquality/sonar/openapi/checks/format/AbstractNamingConventionCheck.java

```diff
@@ -10,7 +10,7 @@ public abstract class AbstractNamingConventionCheck extends BaseCheck {
 	
 	private static final String CAMEL_REGEX = "[a-z]+([A-Z][a-z]+)*([A-Z])?";
 	private static final String SNAKE_REGEX = "^[a-z0-9_$]*$";
-	private static final String KEBAB_REGEX = "^[a-z0-9-]*$";
+	private static final String KEBAB_REGEX = "^[a-z0-9-.]*$";
```

Comment:
This change affects other rules like parameter naming convention check that should be unaffected, this change only applies to kebab case now.

You should declare custom logic in OAR011UrlNamingConventionCheck to be able to accept dots in the expected cases and not in the initial and final position in each path section.